creusot-contracts/src/std/time.rs

```diff
@@ -1,12 +1,15 @@
 use crate::*;
-pub use ::std::time::*;
+pub use ::std::{
+    ops::{Add, Sub},
+    time::*,
+};
 
 impl ShallowModel for Duration {
     type ShallowModelTy = Int;
 
     #[logic]
     #[trusted]
-    #[ensures(result >= 0 && result <= @u128::MAX)]
+    #[ensures(result >= 0 && result <= secs_to_nanos(@u64::MAX) + 999_999_999)]
```

Comment:
I kind of wonder if it wouldn't make sense to have a wrapper around `Int` that provides these methods... idk.

Reply:
Something like `struct DurationNano(Int)`?

Reply:
Yea exactly, idk if it's worth the hassle

Reply:
Yeah maybe a bit overkill, but I could factor the constant out into a function (a constant wouldn't work I assume)?

Reply:
Just leave it for now, it's fine like this. A constant should work, we just don't support const functions or const generics